examples/postgres/migrations/20180522_example_up.sql

```diff
@@ -0,0 +1,20 @@
+/*
+	Migration file generated by protoc-gen-orm. DO NOT EDIT.
+	source: ormpb/examples/postgres/example.proto
+*/
+
+
+/* Generated for Example */
+CREATE TABLE examples (
```

Comment:
can create some tests that actually boots up an mysql/psql and run those migrations against that?